core/src/telemetry/mod.rs

```diff
@@ -2,9 +2,7 @@ pub mod otlp;
 
 use crate::types::Origin;
 
-pub const ATTRIBUTE_MULTIADDRESS: &str = "multiaddress";
 pub const ATTRIBUTE_OPERATING_MODE: &str = "operating_mode";
```

Comment:
Do you think operation mode auto switch can be a source of issues in the future?

Reply:
It can cause duplicates by name, but it is supposed to work like that, and delta would be zero for switched value. So, I don't know, but I don't see it will be big issue, and we want to redesign it anyway.